docs/overview/office-add-in-availability.md

```diff
@@ -19,15 +19,15 @@ To work as expected, your Office Add-in might depend on a specific Office applic
](../reference/requirement-sets/excel-api-1-1-requirement-set.md)<br>- [ExcelApi 1.2](../reference/requirement-sets/excel-api-1-2-requirement-set.md)<br>- [ExcelApi 1.3](../reference/requirement-sets/excel-api-1-3-requirement-set.md)<br>- [ExcelApi 1.4](../reference/requirement-sets/excel-api-1-4-requirement-set.md)<br>- [ExcelApi 1.5](../reference/requirement-sets/excel-api-1-5-requirement-set.md)<br>- [ExcelApi 1.6](../reference/requirement-sets/excel-api-1-6-requirement-set.md)<br>- [ExcelApi 1.7](../reference/requirement-sets/excel-api-1-7-requirement-set.md)<br>- [ExcelApi 1.8](../reference/requirement-sets/excel-api-1-8-requirement-set.md)<br>- [ExcelApi 1.9](../reference/requirement-sets/excel-api-1-9-requirement-set.md)<br>- [ExcelApi 1.10](../reference/requirement-sets/excel-api-1-10-requirement-set.md)<br>- [ExcelApi 1.11](../reference/requirement-sets/excel-api-1-11-requirement-set.md)<br>- [ExcelApi 1.12](../reference/requirement-sets/excel-api-1-12-requirement-set.md)<br>- [ExcelApi 1.13](../reference/requirement-sets/excel-api-1-13-requirement-set.md)<br>- [ExcelApi 1.14](../reference/requirement-sets/excel-api-1-14-requirement-set.md)<br>- [DialogApi 1.1](../reference/requirement-sets/dialog-api-requirement-sets.md)<br>- [IdentityAPI 1.3](../reference/requirement-sets/identity-api-requirement-sets.md)<br>- [ImageCoercion 1.1](../reference/requirement-sets/image-coercion-requirement-sets.md#imagecoercion-11)<br>- [ImageCoercion 1.2](../reference/requirement-sets/image-coercion-requirement-sets.md#imagecoercion-12)<br>- [OpenBrowserWindowApi 1.1](../reference/requirement-sets/open-browser-window-api-requirement-sets.md)<br>- [RibbonApi 1.1](../reference/requirement-sets/ribbon-api-requirement-sets.md)<br>- [SharedRuntime 1.1](../reference/requirement-sets/shared-runtime-requirement-sets.md)|- [BindingEvents](../reference/requirement-sets/office-add-in-requirement-sets.md#bindingevents)<br>- [CompressedFile](../reference/requirement-sets/office-add-in-requirement-sets.md#compressedfile)<br>- [DocumentEvents](../reference/requirement-sets/office-add-in-requirement-sets.md#documentevents)<br>- [File](../reference/requirement-sets/office-add-in-requirement-sets.md#file)<br>- [MatrixBindings](../reference/requirement-sets/office-add-in-requirement-sets.md#matrixbindings)<br>- [MatrixCoercion](../reference/requirement-sets/office-add-in-requirement-sets.md#matrixcoercion)<br>- [Selection](../reference/requirement-sets/office-add-in-requirement-sets.md#selection)<br>- [Settings](../reference/requirement-sets/office-add-in-requirement-sets.md#settings)<br>- [TableBindings](../reference/requirement-sets/office-add-in-requirement-sets.md#tablebindings)<br>- [TableCoercion](../reference/requirement-sets/office-add-in-requirement-sets.md#tablecoercion)<br>- [TextBindings](../reference/requirement-sets/office-add-in-requirement-sets.md#textbindings)<br>- [TextCoercion](../reference/requirement-sets/office-add-in-requirement-sets.md#textcoercion)|
```

Comment:
Did 1.14 make the 2021 cut off? I had assumed it hadn't when we updated the docs elsewhere. If 1.14 made the cut, please edit the line "Office 2021 supports ExcelApi 1.13 and earlier." on this page https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets

Reply:
Confirmed that Office 2021 supports ExcelApi 1.14. I'll make the change on the page you linked.